packages/bitcore-wallet-service/package.json

```diff
@@ -28,6 +28,7 @@
     "bitcore-lib-cash": "^8.17.0",
     "body-parser": "^1.11.0",
     "compression": "^1.6.2",
+    "bitcore-wallet-client": "^8.17.0",
```

Comment:
no, please remove BWC deps from BWS.

BWC already depends on BWS for testing.

Reply:
we could move them to CWC.

Reply:
That seems backwards to me, but I can remove it.

I would normally have the client required in the server.